pkgs/servers/nosql/eventstore/default.nix

```diff
@@ -1,72 +1,79 @@
-{ lib, stdenv
-, fetchFromGitHub
-, fetchurl
-, makeWrapper
+{ lib
+, git
 , dotnetCorePackages
+, glibcLocales
+, buildDotnetModule
+, fetchFromGitHub
+, bintools
+, stdenv
 , mono
-, Nuget
+, patchelf
 }:
 
-let
-
-  dotnet-sdk = dotnetCorePackages.sdk_5_0;
-
-  deps = import ./deps.nix { inherit fetchurl; };
-
-in
-
-stdenv.mkDerivation rec {
-
+buildDotnetModule rec {
   pname = "EventStore";
-  version = "5.0.8";
+  version = "21.10.5";
 
   src = fetchFromGitHub {
     owner = "EventStore";
     repo = "EventStore";
     rev = "oss-v${version}";
-    sha256 = "021m610gzmrp2drywl1q3y6xxpy4qayn580d855ag952z9s6w9nj";
+    sha256 = "sha256-uUDjTGCiQgXmvOUsujIA0JkGQGuw9U4zLKDP1WIFq1o=";
+    leaveDotGit = true;
```

Comment:
Why do we need this?

Reply:
`EventStore` during build executes scripts that look into `.git` to determine the application version and build commit. Without a `.git` folder, they fail and the entire build fails. It's also a bit hard to patch it out, due to it being pretty integrated into the build.

Reply:
Could we hardcode that to a version with fake-git? leaving .git in is not great for reproducibility.

Reply:
It's possible, but the GitInfo package used by eventstore calls `git` *a lot* - https://github.com/devlooped/GitInfo/blob/main/src/GitInfo/build/GitInfo.targets

I could add it if there was a generic way to fake git, but since I didn't find it, it's a lot simpler (and more robust) to simply use the real `git` binary.